Automattic/jetpack
# Manage: Deprecate as a module, bake it directly in.

Manage is an integral part of Jetpack, so much so that we've been auto-activating and hiding the UI to disable it for over a year.  

This PR is an attempt at baking in the Manage functionality so that it's always working if the site is connected.  Most of the code removed was UI around enabling it.  The only pieces that required "baking in" seem to be the [url rerouting](https://github.com/Automattic/jetpack/compare/update/manage-baked-in?expand=1#diff-9ba9fa5bb52bd6cf36c6c0bdd5ed4830R596), and [validating JSON API requests](https://github.com/Automattic/jetpack/compare/update/manage-baked-in?expand=1#diff-59094747abbd58bfafac8e9a099e2fd0R61).  

It really feels like I'm missing something here, so I would love for someone who worked on Manage to take a look.  

To Test: 
- Make sure you're able to do everything you used to be able to in regards to interacting with your site from calypso.  

I did leave the JITM code for manage in, but removed any mention of the manage module itself and the links to enable it.  Instead, the JITMs would just mention something about how awesome autoupdates are and to go to wp.com to enable them for your plugins.  Though currently we're not showing these JITMs anyway.  

Fixes #8754
